src/main/webapp/inc.erzeugeFormular.jsp

```diff
@@ -130,5 +130,6 @@
 <%@ include file="forms/noarray.sqlselect.jsp" %>
 <%@ include file="forms/noarray.textarea.jsp" %>
 <%@ include file="forms/noarray.textfield.jsp" %>
+<%@ include file="forms/noarray.gnd.jsp" %>
```

Comment:
Since the GND field is rendered via `noarray.textfield.jsp` and the new template is just for handling the GND links, i think we should rename it to `noarray.gndlink.jsp` (and also change the DB entry for it.)

Reply:
ich habe es umgenannt, auch in inc.erzeugeFormular.jsp namen ersetzt, was oder wo : (and also change the DB entry for it.) ?